rubrik_cdm/data_management.py

```diff
@@ -192,12 +192,13 @@ def on_demand_snapshot(self, object_name, object_type, sla_name='current', files
 
         return (api_request, snapshot_status_url)
 
-    def object_id(self, object_name, object_type, host_os=None, timeout=15):
+    def object_id(self, object_name, object_type, host_os=None, host_name=None, timeout=15):
         """Get the ID of a Rubrik object by providing its name.
 
         Arguments:
             object_name {str} -- The name of the Rubrik object whose ID you wish to lookup.
             object_type {str} -- The object type you wish to look up. (choices: {vmware, sla, vmware_host, physical_host, fileset_template, managed_volume, aws_native, vcenter})
+            host_name {str} -- Valid and required for oracle_db object type. This is the (or one of the) host names the database is running on.
```

Comment:
`The hostname, or one of the hostnames in the cluster, that the Oracle database is running. Required when the object_type is oracle_db` ... I think that reads a little better. I'm also assuming what you meant by "or one of the" was related to a cluster here.